packages/next-auth/src/core/pages/signin.tsx

```diff
@@ -101,8 +102,28 @@ export default function SigninPage(props: SignInServerPageParams) {
                 {callbackUrl && (
                   <input type="hidden" name="callbackUrl" value={callbackUrl} />
                 )}
-                <button type="submit" className="button">
-                  Sign in with {provider.name}
+                <button
+                  type="submit"
+                  className="button"
+                  style={
+                    {
+                      "--provider-bg": provider.style?.bg ?? "",
```

Comment:
Let's use default values, so we can remove them in the default provider configs.

Reply:
Yeah so theres default values set in the CSS already actually.

![image](https://user-images.githubusercontent.com/7415984/201631118-ed76fba6-1034-49a5-a172-7c62ff1bd27f.png)